src/poetry/json/schemas/poetry.json

```diff
@@ -39,7 +39,16 @@
         },
         "secondary": {
           "type": "boolean",
-          "description": "Declare this repository as secondary, i.e. it will only be looked up last for packages."
+          "description": "Declare this repository as secondary, i.e. default repositories take precedence."
+        },
+        "priority": {
+          "enum": [
+            "primary",
+            "default",
+            "secondary",
+            "explicit"
+          ],
+          "description": "Declare the priority repository of this repository."
```

Comment:
I think this should be `Declare the priority repository of this repository.`

Reply:
Indeed, thank you for pointing that out!